docs/sources/http_api/dashboard_versions.md

```diff
@@ -9,6 +9,8 @@ aliases = ["/docs/grafana/latest/http_api/dashboardversions/"]
 
 ## Get all dashboard versions
 
+> **Warning:** This API is deprecated since Grafana v9.0.0 and will be removed in a future release. Refer to the [new dashboard versions API](#get-all-dashboard-verions-by-dashboard-uid).
```

Comment:
this link does not work because of a misspelling
```suggestion
> **Warning:** This API is deprecated since Grafana v9.0.0 and will be removed in a future release. Refer to the [new dashboard versions API](#get-all-dashboard-versions-by-dashboard-uid).
```